src/i18n/es.js

```diff
@@ -20,6 +20,10 @@ export const messages_es = {
   small: `Pequeño`,
   social_networks: `Redes sociales`,
   social_share_title: 'Compartir',
+  'tabs.audit.rules.payu_button.empty_links.description':
+    'Los botones de pago PayU no funcionarán cuando el usuario los pulse si no tienen enlace de solicitud de pago. Añade un enlace a los siguientes botones.',
```

Comment:
No estoy seguro, pero me suena mejor:

```suggestion
    'Los botones de pago PayU no funcionarán cuando el usuario los pulse si no tienen enlace. Añade un enlace  de solicitud de pago a los siguientes botones.',
```

Reply:
Ya lo escale a productos aquí... https://makingsense.slack.com/archives/C023YV1B7B6/p1669302646310939

Reply:
No nos bloqueemos, elegí la opción que te parezca más adecuada y continuemos.